Challenge-4/Bot_2/src/constants.ts

```diff
@@ -0,0 +1,55 @@
+import { ethers } from "forta-agent";
+import { BotConfig } from "./types";
+
+export const CONFIGURATION_ABI = [
+  "function getConfiguration(address cometProxy) view returns (tuple(address governor, address pauseGuardian, address baseToken, address baseTokenPriceFeed, address extensionDelegate, uint64 supplyKink, uint64 supplyPerYearInterestRateSlopeLow, uint64 supplyPerYearInterestRateSlopeHigh, uint64 supplyPerYearInterestRateBase, uint64 borrowKink, uint64 borrowPerYearInterestRateSlopeLow, uint64 borrowPerYearInterestRateSlopeHigh, uint64 borrowPerYearInterestRateBase, uint64 storeFrontPriceFactor, uint64 trackingIndexScale, uint64 baseTrackingSupplySpeed, uint64 baseTrackingBorrowSpeed, uint104 baseMinForRewards, uint104 baseBorrowMin, uint104 targetReserves, tuple(address asset, uint8 decimals, uint256 conversionFactor)[] assetConfigs) configuration)",
```

Comment:
For legibility, you should make this a multi-line string.

Reply:
Could make it so you don't need the numbered comments from below and put each return type in the tuple in its own line (or a couple per line), or something that just makes it easier to read. 🙂